backend/scrapers/classes/parsers/ellucianSectionParser.js

```diff
@@ -176,9 +176,21 @@ class EllucianSectionParser extends ellucianBaseParser.EllucianBaseParser {
       retVal.maxCredits = creditsParsed.maxCredits;
       retVal.minCredits = creditsParsed.minCredits;
     } else {
-      macros.log('warning, nothing matchied credits', url, text);
+      macros.log('warning, nothing matched credits', url, text);
     }
 
+    // grabs optional feees
+    const table = $('body > div.pagebodydiv > table > tr > td > table');
+
+    // for all tables... if table includes word "amount", parse table, get fees/description
+    for (let i = 0; i < table.length; i++) {
+      if ($(table[i]).attr().summary.includes('amount')) {
+        const parsedTable = this.parseTable(table[i]).tableData;
+
+        retVal.feeDescription = parsedTable.description[0];
```

Comment:
NEU's site can be oddddd at times. Can you add the following checks?

 - A check to make sure parsedTable.description.length > 0 and parsedTable.amount.description.length > 0. If not, logs a warning and doesn't parse any fees from this page. 

 - a check to see if parsedTable.amount.length > 1 ||  parsedTable.description.length > 1 and if so, logs a warning (multiple fees?)

Reply:
Oh also, can we should parse the value into a number instead of a string? And can we pass it to the rest of the codebase as a number instead of a string too? 

And one more check:
 - If it isn't a number for whatever reason, log a warning and don't save any fees